Enumerables.rb

```diff
@@ -0,0 +1,70 @@
+module Enumerable
+    def my_each
+        i = 0
+        while i<self.size
+            yield self[i]
+            i+=1
+        end
+    end
+
+    def my_each_with_indeitem
+        i = 0
+        result = []
```

Comment:
[OPTIONAL] The default method of ruby is each_with_index. For better readability let's rename it to my_each_with_index.

Reply:
I think this is autocompletion mistake I was not intending to name like that